### ISSUE TITLE
Icon Requests: Riddles

### ISSUE THREAD

**Issue by db0** [game-icons/icons#843]
- [ ] Enimga
- [ ] Riddle
- [ ] Question Mark
- [ ] Rubic Cube
- [ ] Gordian Knot

**Comment by blawnode** [comment]
I agree! Especially about the question mark - This one is handy for many things, and goes nicely in the GUI section, too.
